run-tests.php

```diff
@@ -450,13 +450,8 @@ function save_or_mail_results()
 $pass_option_n = false;
 $pass_options = '';
 
-$compression = 0;
 $output_file = $CUR_DIR . '/php_test_results_' . date('Ymd_Hi') . '.txt';
 
-if ($compression && in_array("compress.zlib", stream_get_filters())) {
-	$output_file = 'compress.zlib://' . $output_file . '.gz';
-}
-
```

Comment:
If you look few lines higher, `$compression` variable is always `0`, so this code is basically dead.